content/en/docs/azure/deploy/install-kubeflow.md

```diff
@@ -101,13 +101,14 @@ Run the following commands to set up and deploy Kubeflow.
   [Kubeflow releases
   page](<https://github.com/kubeflow/kfctl/releases/tag/>{{% kf-latest-version %}}).
 
-1. Unpack the tar ball
+1. Unpack the tar ball:
 
     ```
     tar -xvf kfctl_{{% kf-latest-version %}}_<platform>.tar.gz
     ```
 
-1. Run the following commands to set up and deploy Kubeflow. The code below includes an optional command to add the binary kfctl to your path. If you don’t add the binary to your path, you must use the full path to the kfctl binary each time you run it.
+1. Run the following commands to set up and deploy Kubeflow. The code below includes an optional command to add the
+   binary   kfctl to your path. If you don’t add the binary to your path, you must use the full path to the kfctl binary each time you run it.
```

Comment:
Can you check number of spaces here please:

```suggestion
   binary kfctl to your path. If you don’t add the binary to your path, you must use the full path to the kfctl binary each time you run it.
```

Reply:
fixed!

Reply:
Line 3: I think we had a similar thing with IBM Cloud's guide. Maybe it would be better to mention that you're deploying KF with Azure CLI or on Azure

```
- description = "Instructions for deploying Kubeflow with the shell"
+ description = "Instructions for deploying Kubeflow on Azure"
```

WDYT?

Reply:
fixed